project/AppDependencies.scala

```diff
@@ -1,24 +1,22 @@
 import sbt.*
 
 object AppDependencies {
-  val bootstrapVersion = "7.23.0"
 
-  val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-28"     % "8.5.0",
-    "uk.gov.hmrc"       %% "play-conditional-form-mapping"  % "1.13.0-play-28",
-    "uk.gov.hmrc"       %% "domain"                         % "8.3.0-play-28",
-    "uk.gov.hmrc"       %% "bootstrap-frontend-play-28"     % bootstrapVersion
+  private val playBootstrapVersion = "8.5.0"
+
+  private lazy val compile = Seq(
+    "uk.gov.hmrc"         %% "play-frontend-hmrc-play-30"             % "8.5.0",
```

Comment:
Is this the same number as `playBootstrapVersion` by coincidence or should it be using the variable?

Reply:
Apparently 8.6.0 is out now as well, but not sure if we should be migrating to that yet.

Reply:
it is a coincidence play-frontend have seprate stories up next, to upgrade to latest 9.10+, 

dont see a big deal to increase from 8.5 to 8.6 as it only provides default application-logger.xml https://github.com/hmrc/bootstrap-play/pull/131/files , let me know if you want I can upgrade